WcaOnRails/app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -176,7 +176,10 @@ def index_for_group_type
     roles = filter_roles_for_logged_in_user(roles)
 
     # Filter the list based on the other parameters.
-    roles = filter_roles_for_parameters(roles: roles, status: params[:status], is_active: params[:isActive])
+    roles = filter_roles_for_parameters(
+      roles: roles, status: params[:status],
+      is_active: params.key?(:isActive) ? ActiveRecord::Type::Boolean.new.cast(params.require(:isActive)) : nil
```

Comment:
Duplicate from translators PR

Reply:
Yes, both the PRs needed this. Once any of them merges, I'll rebase the other one and probably gets rebased without any conflict as the changes are same.
I've made the same change I've done in the other PR here.

Reply:
Other PR is merged :wink

Reply:
Thanks, rebased and confirmed that this file disappeared from the change list :-)